migration/README.md

```diff
@@ -0,0 +1,47 @@
+# Database Schema Migration
+
+For the database schema migration, we use the [Atlas tool](https://github.com/ariga/atlas).
+
+When you install the Bucketeer Helm application, the migration runs automatically before the Bucketeer application is installed.
+If the migration fails, it won't install the application.
+
+## Prerequisite
+
+Ensure that you have started the Minikube in the dev container and that the `localenv-mysql-0` pod is running without errors.
```

Comment:
It would be helpful to have a link to instructions that explain how to run minikube on devcontainer.